src/azure-cli/azure/cli/command_modules/acr/custom.py

```diff
@@ -272,7 +272,7 @@ def acr_login(cmd,
             username=username,
             password=password)
 
-        logger.warning("You can perform manual login using the provided access token below, "
+        logger.info("You can perform manual login using the provided access token below, "
```

Comment:
Running flake8 on modules...
> ERROR: /mnt/vss/_work/1/s/src/azure-cli/azure/cli/command_modules/acr/custom.py:276:24: E127 continuation line over-indented for visual indent
/mnt/vss/_work/1/s/src/azure-cli/azure/cli/command_modules/acr/custom.py:276:24: E127 continuation line over-indented for visual indent
2     E127 continuation line over-indented for visual indent

Please fix this linter issue